packages/server/ts/map/regions.ts

```diff
@@ -37,10 +37,10 @@ class Regions {
     }
 
     loadDoors() {
-        let doors = map.doors;
+        const doors = map.doors;
```

Comment:
I like it. That needs be our rule in the project - if a variable is not going to change and shouldn't change then let's use `const`. We will get a ts warning when trying to mutate it

Reply:
It already is added.
https://eslint.org/docs/rules/prefer-const